app/controllers/standup_meetings_controller.rb

```diff
@@ -0,0 +1,20 @@
+class StandupMeetingsController < ApplicationController
+  before_action :set_standup_meeting_group
+
+  def edit
+    @standup_meeting = StandupMeeting.find(params[:id])
+    # TODO: A naïve date range with a single min and max is probably fine for now,
+    # but there is an islands and gaps problem that should be handled to address
+    # issues when a user joins a group, leaves the group, and rejoins.
+    @date_range = [{
+      from: StandupMeeting.for_member(current_user, @standup_meeting_group).minimum(:meeting_date),
+      to: StandupMeeting.for_member(current_user, @standup_meeting_group).maximum(:meeting_date)
+    }]
```

Comment:
I am kind of confused why we need this and what the flow is for editing a standup meeting?

Reply:
My recollection during the standup for pair app, you and I discussed that a standup meeting will be created each day for current standup groups through a cron task. I took this to mean that standup meetings should only be available for the days in which you've joined a standup meeting group.

If this is the case, the date range helps us restrict edge cases wherein a user tries to create a standup meeting for a date he wasn't a member of the group. Does that make sense?